pkg/storaged/stratis-details.jsx

```diff
@@ -706,27 +706,28 @@
                              content={content} />;
 };
 
-export function unlock_pool(client, uuid, show_devs) {
+export function start_pool(client, uuid, show_devs) {
     const manager = client.stratis_manager;
-    const locked_props = manager.LockedPools[uuid];
-    const devs = locked_props.devs.v.map(d => d.devnode).sort();
-
-    if (!locked_props.key_description ||
-        locked_props.key_description.t != "(bv)" ||
-        !locked_props.key_description.v[0] ||
-        locked_props.key_description.v[1].t != "(bs)" ||
-        !locked_props.key_description.v[1].v[0]) {
-        dialog_open({
-            Title: _("Error"),
-            Body: _("This pool can not be unlocked here because its key description is not in the expected format.")
-        });
-        return;
+    const stopped_props = manager.StoppedPools[uuid];
+    const devs = stopped_props.devs.v.map(d => d.devnode).sort();
+    let key_desc = null;
+
+    if (stopped_props.key_description &&
+        stopped_props.key_description.t == "(bv)" &&
+        stopped_props.key_description.v[0]) {
+        if (stopped_props.key_description.v[1].t != "(bs)" ||
+            !stopped_props.key_description.v[1].v[0]) {
+            dialog_open({
+                Title: _("Error"),
+                Body: _("This pool can not be unlocked here because its key description is not in the expected format.")
+            });
+            return;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18679-20230420-131050-f1115ac6-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)